mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/ServerCmd.scala

```diff
@@ -1,17 +1,12 @@
 package at.forsyte.apalache.tla.tooling.opt
 
-import java.io.File
-
 import com.typesafe.scalalogging.LazyLogging
 import at.forsyte.apalache.shai
 
 class ServerCmd
     extends ApalacheCommand(name = "server", description = "Run in server mode (in early development)")
     with LazyLogging {
 
-  // Dummy file used for application (esp. output) configuration
-  def file = new File("server")
```

Comment:
This dummy file is no longer needed now that the `OutputManager` can fall back to the `routine` config value.